geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,271 @@
+from __future__ import absolute_import, division
+
+import itertools
+import warnings
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+from shapely.geometry import Point, Polygon, LineString
+
+from geopandas import GeoSeries, GeoDataFrame, plotting
+from geopandas.tests.test_plotting import _check_colors
+
+import pytest
+
+@pytest.fixture(autouse=True)
+def close_figures(request):
+    yield
+    plt.close('all')
+
+try:
+    cycle = mpl.rcParams['axes.prop_cycle'].by_key()
+    MPL_DFT_COLOR = cycle['color'][0]
+except KeyError:
+    MPL_DFT_COLOR = matplotlib.rcParams['axes.color_cycle'][0]
+
+class TestCMapPointPlotting:
+    def setup_method(self):
+        # Intializing the dataframe
+        self.N = 10
+        self.points = GeoSeries(Point(i, i) for i in range(self.N))
+        self.values = np.arange(self.N)
+        self.df = GeoDataFrame({'geometry': self.points, 'values': self.values})
+
+    def test_customnorm(self):
+        # test color without additional parameters
+        from geopandas.plotting import plot_point_collection
+        fig, ax = plt.subplots()
+        coll = plot_point_collection(ax, self.points, self.values)
+        fig.canvas.draw_idle()
+
+        cmap = plt.get_cmap()
```

Comment:
you should probably be getting this directly from the collection `coll.get_cmap()`